products/BellHybrid/alarms/include/popups/BedtimeReminderPopupRequestParams.hpp

```diff
@@ -10,7 +10,7 @@ namespace gui
     class BedtimeReminderPopupRequestParams : public PopupRequestParams
     {
       public:
-        BedtimeReminderPopupRequestParams() : PopupRequestParams{gui::popup::ID::BedtimeNotification}
+        BedtimeReminderPopupRequestParams() : PopupRequestParams{gui::popup::ID::BedtimeNotification, {popup::Disposition::Priority::High, popup::Disposition::WindowType::Popup}}
```

Comment:
are you sure that it's required? I'm asking if it's not handled by fallbacks.

Reply:
yep, that was unnecessary step, reverting that part